Cartfile

```diff
@@ -1,2 +1,2 @@
-github "mapbox/mapbox-vision-ios" == 0.0.1-alpha.4
```

Comment:
Change after https://github.com/mapbox/mapbox-vision-ios/pull/25

Reply:
We may release `MapboxVision` v0.0.1-alpha.4.1` and then reference this version here.

Reply:
Ok, alpha-4.1 is released.